src/libtools/calculator.cc

```diff
@@ -18,151 +18,7 @@
 // Local VOTCA includes
```

Comment:
Do we still need this file?

Reply:
Not really, but on the other hand it is always good to not have to create it again and it does no harm, so I would leave it.